src/pages/mesh/basic/create-mesh.md

```diff
@@ -77,11 +81,25 @@ When creating or updating a mesh, the file to upload must have the `.json` filen
 
 1. When you are prompted to confirm that you want to create a mesh, select **Yes**. If you want to automatically confirm the creation, add the `-c` or `--autoConfirmAction` flag to your create command.
 
-  The `aio api-mesh:create` response assigns you a `meshId`. Use the [`aio api-mesh:status`](../advanced/index.md#aio-api-meshstatus) command to see the status of your mesh creation. You can run the [`aio api-mesh:describe`](../advanced/index.md#aio-api-meshdescribe) command to get your `apiKey` and a GraphQL endpoint that you can use to query your mesh.
+### Access your mesh URLs
+
+The `aio api-mesh:create` response assigns you a `meshId`. Use the [`aio api-mesh:status`](../advanced/index.md#aio-api-meshstatus) command to see the status of your mesh creation. You can run the [`aio api-mesh:describe`](../advanced/index.md#aio-api-meshdescribe) command to get your `apiKey` and a GraphQL endpoint that you can use to query your mesh.
+
+Until Adobe deprecates legacy mesh URLs, the response to the `aio api-mesh:describe` command will list both the legacy and edge mesh URLs. The legacy mesh URL is a standard mesh URL, which Adobe will deprecate later this year. The edge mesh URL offers improved response times. API Mesh stores edge meshes closer to your data sources to reduce latency and generally improve the API Mesh experience by making meshes more performant. Edge meshes provide the following advantages:
+
+- Improved response times and enhanced API performance
+- Easier integration with industry-standard tools and third-party products
+- Reduced potential of other hosted meshes degrading the performance of your mesh
```

Comment:
Can we change this to "Enhanced performance reliability through inherent tenant isolation"